webviews/src/views/RokuFileSystemView/FileSystemEntry.svelte

```diff
@@ -17,6 +17,10 @@
         dispatch('open', entry);
     }
 
+    function onDeleteClick() {
+        dispatch('delete', entry);
```

Comment:
I'm thinking we should add a confirmation dialog before executing the delete? Might need to do it in the vscode side?

Reply:
I personally wouldn't just for how I'm using it:
- I'm deleting 10-20 files from cachefs, to get my app to recache things. I'm also not deleting everything, so a button to delete all of cachefs is not the answer (although it could be a good addition in a future PR - I didn't add that yet since there was no equivalent for it in ODC). A pop-up per file would probably irritate me and I wouldn't use this
- You'd mostly use this to delete tmp or cachefs, which is by no means critical if deleted accidentally. I guess there's the case for additional volumes, but that's way less common

Reply:
I mostly agree that it's likely not mission critical. @TwitchBronBron @iBicha maybe a compromise is a setting to turn off the confirmation?

Reply:
Okay. If it's not worth it, no need adding complexity.

Reply:
Let me know what's a nice way to proceed
- No pop-up
- Pop-up each time
- Pop-up with a note that there's a vscode setting to disable it

Reply:
We can ship without a popup. If someone wants that, they can ask or implement it. :)